objstore.go

```diff
@@ -435,6 +435,12 @@ func WrapWithMetrics(b Bucket, reg prometheus.Registerer, name string) *metricBu
 			Name: "objstore_bucket_last_successful_upload_time",
 			Help: "Second timestamp of the last successful upload to the bucket.",
 		}, []string{"bucket"}),
+
+		opsUploadedBytes: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name:        "objstore_bucket_operation_uploaded_bytes_total",
```

Comment:
```suggestion
			Name:        "objstore_bucket_operation_written_bytes_total",
```

We have two options. Either objstore_bucket_operation_written_bytes_total and make sure we increment every time we write (upload) some bytes to bucket. Technically the only significant bytes are written in Upload method, but we don't know in future, so let's keep operation label and put "write" in name so we are flexible here.